src/utils/constants.js

```diff
@@ -103,7 +105,21 @@ export const ALERT_MESSAGE = {
   [ALERT_ERROR_UNEXPECTED]: 'Therer was an unexpected error.',
 };
 
-export const ALERT_MESSAGE_DURATION = 3000;
```

Comment:
Please restore this as this is required for Snackbar.

Reply:
Very odd this wasn't uploaded properly. Was one of the first errors I had to fix since it would not compile without. Currently has statement.

Reply:
Okay, so upon further investigation the push is severely outdated since I forgot to do a commit. However, I should address many of these comments before another push is made.